src/components/Contacts/Contacts.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styles from './Contacts.module.css';
+
+const Contacts = () => (
+    <div className={styles.wrap}>
+        <h1>Контакты</h1>
+    </div>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)